backend/src/routes/MeetingRoutes.ts

```diff
@@ -0,0 +1,19 @@
+import { Router } from 'express';
+
+import {
+    getAllMeetings,
+    getMeeting,
+    addMeeting,
+    deleteMeeting,
+    updateMeeting,
+} from '../controllers/MeetingController';
+
+const MeetingRouter = Router();
+
+MeetingRouter.route('/').get(getAllMeetings).post(addMeeting);
+MeetingRouter.route('/:meetingId')
+    .get(getMeeting)
+    .delete(deleteMeeting)
+    .patch(updateMeeting);
```

Comment:
In order to access the `req.body.requester` you'll need to add the `protect` middleware before all your endpoints.

As Lauren is currently still working on the permission system, just leave it without any parameters, e.g: `.get(protect(), getAllMeetings)`